FixedBGExamples/BoostedBHComplexScalar/params.txt

```diff
@@ -0,0 +1,101 @@
+#Params for runtime inputs
+
+verbosity = 0
+chk_prefix = /cosma6/data/dp174/dc-tray2/FixedBGRComplex/mu005/v03_chk_
+plot_prefix = /cosma6/data/dp174/dc-tray2/FixedBGRComplex/mu005/v03_plt_
+#restart_file = 
+
+# Set up grid spacings and regrid params
+# NB - the N values need to be multiples of block_factor
+N1 = 128
+N2 = 64
+N3 = 64
+
+# Length of longest box side, dx_coarsest = L/N(max)
+# NB if non cubic, this determines dx for all sides
+L = 4096
+
+# Regridding
+tag_buffer_size = 0
+#regrid_length = 2048
```

Comment:
Maybe get rid of this regrid_length param as I think we did not use it in the end